coordinator/src/services/translation/request/organization.ts

```diff
@@ -26,11 +26,11 @@ export function convertOrganizationAndProviderLicense(
   return hl7V3Organization
 }
 
-function convertRepresentedOrganization(
+export function convertRepresentedOrganization(
```

Comment:
according to intellij this is only used in one place in the repo (above in this file) so no need to export